### ISSUE TITLE
Migrate to new playwright version

### ISSUE THREAD

**Issue by mmarkelov** [mmarkelov/jest-playwright#33]
[Playwright 0.11.0](https://github.com/microsoft/playwright/releases/tag/v0.11.0) was released, as I know there were some changes with API, so it can need some changes to current implementation
#30

**Comment by schindld** [comment]
This still isn't working for me. I'm using the "'jest-playwright-preset-example" as my starting point. I updated the *package.json* to include

```json
  "devDependencies": {
    "jest": "^25.1.0",
    "jest-playwright-preset": "mmarkelov/jest-playwright#master",
    "playwright-chromium": "0.11.0"
  },
  "jest": {
    "preset": "jest-playwright-preset"
  }
```
because you merged in support for 0.11.0 this morning.

Next, I update the lock file: `npm install --no-optional --package-lock-only`.

Then I saw you were installing *libgbm* in your workflow file, so after copying the package files into my docker image, I run `apt-get update && apt-get install -y libgbm-dev && npm install`.

The `npm install` fails with **ENOENT: no such file or directory, chmod '/usr/src/app/node_modules/jest-playwright-preset/lib/bin/index.js'**.

So, what am I missing?

**Comment by schindld** [comment]
Looks like you have a build script that has to run in order to get that file in place, so I can't just use master - I have to wait for a release.

**Comment by mxschmitt** [comment]
Exactly, you have to wait for a release 👍

**Comment by mmarkelov** [comment]
@schindld I just released **0.0.10**, sorry but I just hadn't time to do it yesterday. 

> This still isn't working for me. I'm using the "'jest-playwright-preset-example" as my starting point. I updated the _package.json_ to include
> 
> ```json
>   "devDependencies": {
>     "jest": "^25.1.0",
>     "jest-playwright-preset": "mmarkelov/jest-playwright#master",
>     "playwright-chromium": "0.11.0"
>   },
>   "jest": {
>     "preset": "jest-playwright-preset"
>   }
> ```

I suppose I can add actual build to git for this case

**Comment by schindld** [comment]
Thanks @mmarkelov 
I love the project
